BismarkUnchained/scenes/Enemy/Enemy.gd

```diff
@@ -7,9 +7,9 @@ func damage():
 
 func _process(delta):
 	var pos = get_global_position()
-	var player_pos = get_node("../../Player/KinematicBody2D").get_global_position()
+	var player_pos = get_node("../../Player/Player").get_global_position()
```

Comment:
this shouldn't be `/Player/Player`, we should name this better right

Reply:
Yes probably. Maybe player/playerbody or something